symfony/client/src/orangehrmAuthenticationPlugin/pages/Login.vue

```diff
@@ -69,6 +69,21 @@
             type="submit"
           />
         </oxd-form-actions>
+        <oxd-text
+          :style="{
+            display: 'flex',
+            justifyContent: 'center',
+            marginTop: '0.5rem',
+          }"
+        >
+          Forgot Your &nbsp;
```

Comment:
change to
```suggestion
          Forgot Your&nbsp;
```
otherwise there are two spaces between "Your" and "Password"